cdm/src/main/java/ucar/nc2/ft2/coverage/HorizCoordSys.java

```diff
@@ -61,81 +56,88 @@
  * @author caron
  * @since 7/11/2015
  */
+// TODO: This class is trying to do too much: it's handling projection, latLon1D AND latLon2D CRSs. We should break
```

Comment:
This is a good idea - can you open an issue for this, tagged with netCDF-java beta as the milestone?